src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/config.jelly

```diff
@@ -13,6 +13,10 @@
     <f:textbox/>
   </f:entry>
 
+  <f:entry field="inheritFrom" title="The name of the pod template to inherit from">
```

Comment:
I think this needs some help text in help-inheritFrom.html, and some docs in the README